examples/capi-quick-start/aws-cluster-calico-crs.yaml

```
@@ -28,8 +28,14 @@ spec:
             strategy: ClusterResourceSet
           cpi: {}
           csi:
+            defaultStorage:
+              providerName: aws-ebs
+              storageClassConfigName: aws-ebs-sc
             providers:
             - name: aws-ebs
+              storageClassConfig:
+              - name: aws-ebs-sc
```

Maybe without the "sc"?
```suggestion
              - name: aws-ebs
```